eng/.docsettings.yml

```diff
@@ -154,6 +154,8 @@ known_presence_issues:
   - ['sdk/streamanalytics/Microsoft.Azure.Management.StreamAnalytics','#5499']
   - ['sdk/graphrbac/Microsoft.Azure.Graph.RBAC','#5499']
   
+  - ['sdk/search','#5499']
```

Comment:
What does this do?

Reply:
It prevents the build from breaking in CI. There is a script that verifies the presence and content of ReadMes.md and also breaks the CI if there are issues with readMe. This lets us by pass it for now till readMes can be updated.

Reply:
Is there a place I can go to learn more about these readme.md files and what they're supposed to do? If I understand correctly, this is a new thing (at least, Search never had one in this repo).

Reply:
This might help [Analyze job](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki?pagePath=%2FThe%20Analyze%20Job%20Failed.%20What%20now%3F&pageId=5&wikiVersion=GBwikiMaster)

Reply:
Thanks!